.coveragerc

```diff
@@ -644,6 +644,7 @@ omit =
     homeassistant/components/livisi/climate.py
     homeassistant/components/livisi/coordinator.py
     homeassistant/components/livisi/switch.py
+    homeassistant/components/livisi/livisi_entity.py
```

Comment:
I also suggest you rename this file to `entity.py`. We already know we're in `livisi` domain...